p2p_wallet/Scenes/Main/Send/SendViaLink/ReceiveFundsViaLink/Coordinator/ReceiveFundsViaLinkCoordinator.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ReceiveFundsViaLinkCoordinator.swift
+//  p2p_wallet
+//
+//  Created by Ivan on 28.03.2023.
+//
+
+import Combine
+import Foundation
+import UIKit
+
+final class ReceiveFundsViaLinkCoordinator: Coordinator<Void> {
+    
+    // Subjects
+    private let resultSubject = PassthroughSubject<Void, Never>()
+    
+    private let presentingViewController: UIViewController
+    private let url: URL
+    
+    // MARK: - Init
+    
+    init(presentingViewController: UIViewController, url: URL) {
+        self.presentingViewController = presentingViewController
+        self.url = url
+    }
+    
+    override func start() -> AnyPublisher<Void, Never> {
+        let viewModel = ReceiveFundsViaLinkViewModel(url: url)
+        let view = ReceiveFundsViaLinkView(viewModel: viewModel)
+        let viewController = view.asViewController()
+        viewController.view.layer.cornerRadius = 16
+        let transition = PanelTransition()
+        transition.containerHeight = 446
```

Comment:
use UIBottomSheetHostingController for dynamic height bottom sheet, so you don't have to clarify container Height and avoid unwanted behavior / layout bug for different devices (you have already work with it in Swap)
https://discord.com/channels/767761912167006292/1083303297437417493